### PR TITLE
[AAE-12134] unit test: Should be able to set number of columns for He…

### PR DESCRIPTION
**Pull request by kathrine0**
…ader widget

**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:

Tests

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-12134

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by pmartinezga — APPROVED** [review]

File: lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts [hunk 1]
```diff
@@ -1056,6 +1057,27 @@ describe('FormCloudComponent', () => {
         });
     });
 
+
```
Comment (eromano) [inline lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts:1060]: double new line

**Review by eromano — DISMISSED** [review]

File: lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts (hunk 1 above)
Comment (eromano) [reply lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts:1060]: how is possible the linter didn't see it?

Comment (kathrine0) [reply lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts]: Good question... anyway I removed that line

Comment (kathrine0) [reply lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts]: created a task for it: https://alfresco.atlassian.net/browse/AAE-12747

**Pull request merged**